tests/test_RemoveLinksToAttachments.py

```diff
@@ -0,0 +1,83 @@
+        conn.commit()
+        test_collectionobjectattachment_data = [("1","a","b","c"),("2","b","c","d"),("4","c","d","e"),("3","d","e","f")]
+        cursor.executemany("""INSERT INTO collectionobjectattachment VALUES (?,?,?,?)""" , test_collectionobjectattachment_data)
+        conn.commit()
+
+        cursor.execute("""CREATE TABLE testdata1 (AttachmentID,data1,data2,data3)""")
+        test_testdata1 = [("1","a","b","c"),("2","b","c","d"),("4","c","d","e"),("3","d","e","f")]
+        cursor.executemany("""INSERT INTO testdata1 VALUES (?,?,?,?)""", test_testdata1)
+        conn.commit()
+
+        cursor.execute("""CREATE TABLE testdata2 (AttachmentID,data1,data2,data3)""")
+        test_testdata2 = [("1", "i", "b", "c"), ("2", "o", "e", "f"), ("2", "g", "h", "i"), ("3", "j", "k", "l")]
+        cursor.executemany(""" INSERT INTO testdata2 VALUES (?,?,?,?)""", test_testdata2)
+        conn.commit()
+
+        cursor.close()
+        conn.close()
+
+    def tearDown(self):
+        """ deletes the test database"""
+        os.remove("specifytest.db")
+
+    def test_select_attachments(self):
+        """ tests to confirm attachmentID's that correspond to a image/jpeg MimeType are being selected properly"""
+        conn = sqlite3.connect("specifytest.db")
+        actual = RemoveLinksToAttachments.select_attachments(conn)
+        expected = [("1",),("2",),("3",)]
+        self.assertEqual(expected,actual)
+
+    def test_delete_attachments_attachment(self):
+        """ tests to confirm that attachmentIDs are deleted with no conflicts properly"""
+        conn = sqlite3.connect("specifytest.db")
+        cursor = conn.cursor()
+        test_referenced_tables = [("testdata1",),("testdata2",),("collectionobjectattachment",)]
+        test_attachments = [("1",),("2",),("3",)]
+        RemoveLinksToAttachments.delete_attachments (conn, test_referenced_tables,test_attachments)
+        cursor.execute("""SELECT * FROM attachment where AttachmentID = 1""")
+        actual = cursor.fetchall()
+        self.assertFalse(actual)
+
+    def test_delete_attachments_no_conflicts(self):
+        """ tests to confirm that when attachments are deleted with no conflicts, corrected values are returned"""
+        conn = sqlite3.connect("specifytest.db")
+        test_referenced_tables = [("testdata1",),("testdata2",),("collectionobjectattachment",)]
+        test_attachments = [("1",),("2",),("3",)]
+        actual = RemoveLinksToAttachments.delete_attachments (conn, test_referenced_tables,test_attachments)
+        expected = []
+        self.assertEqual(expected,actual)
+
+    def test_delete_attachments_conflicts(self):
+        """ tests to confirm that when attachments are deleted and conflicts occur, corrected values are returned"""
+        conn = sqlite3.connect("specifytest.db")
+        test_referenced_tables = [("testdata1",),("testdata",),("collectionobjectattachment",)]
```

Comment:
`sqlite3.OperationalError: no such table: testdata` - did you forget to create this table, or is this a typo in the table name?

Reply:
Test script has been updated so there is no longer an error thrown